students/TinaB/lesson04/katafourteen.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import random
+
+# book_text_file = "sherlock.txt"
+#"sherlock_small.txt"
+text_file = open("sherlock_small.txt", 'r')
+text = text_file.read()
+# first option takes text as is, second
+words_in_text = text.split()
+#words_in_text = text.replace('--', ' ').replace('.', '').replace(',', '').replace(
+#    '"', '').replace("'", '').replace(":", ' ').replace('?', '').replace('!', '').replace('\n', ' ').replace('(', '').replace(')', '').split()
```

Comment:
It's a best practice to remove all commented code from your files before committing to git. I had assumed this assignment was still a WIP partly because of all the commented code.